www/caddy-custom/Makefile

```diff
@@ -41,7 +41,7 @@ XCADDY_ENV+=	XCADDY_WHICH_GO="${GO_CMD}" \
 
 do-build:
 	${MKDIR} ${WRKSRC}
-	cd ${WRKSRC} && ${XCADDY_ENV} ${LOCALBASE}/bin/xcaddy build ${PLUGINS}
+	cd ${WRKSRC} && ${XCADDY_ENV} ${LOCALBASE}/bin/xcaddy build ${CADDY_CUSTOM_VERSION} ${PLUGINS}
```

Comment:
This I don't understand. Are we not building CADDY_VERSION already pinned by www/caddy? So I would assume that version that FreeBSD ships is fine...

Reply:
When reviewing what was happening in the xcaddy build, there was no version passed to it, which means xcaddy pulls the latest caddy version and builds it. So a new release would then be instantly built. This is all network enabled.

I just want to be able to verify a build before it happens automatically. This was a good idea to lock the version down (the xcaddy docs explain it)

Reply:
To be frank that sounds like a bug in the FreeBSD port and it should just use CADDY_VERSION here. It's using that version number for the package anyway.

Reply:
I'll try to log whats happening exactly again tomorrow. I just didnt see a version tag, commit hash or anything else passed after the xcaddy command. And that means (for me) (if there is no voodoo involved I dont yet understand) it will always pull latest, since xcaddy also behaves like that outside of the jail.

Reply:
"<caddy_version> is the core Caddy version to build; defaults to CADDY_VERSION env variable or latest..."

I've logged this CADDY_VERSION env variable as 2.7.6.3 in the build, and that means the xcaddy will adhere to that.

I will remove the custom versioning.